src/class.js

```diff
@@ -19,6 +15,38 @@
 // `meow` that should return the string `<name> meowed!` where `<name>` is the `name`
 // property set on the Cat instance.
 
+class User {
+  constructor(options) {
+    this.email = options.email;
+    this.password = options.password;
+  }
+  comparePasswords(newPasswprd) {
+    if (this.password === newPasswprd) {
```

Comment:
You can one-line this with `return this.password === newPassword;`. Also, you have a typo on `newPassword` here.